R/ttestpairedsamples.R

```diff
@@ -265,6 +273,11 @@ TTestPairedSamples <- function(jaspResults, dataset = NULL, options, ...) {
 
       confIntEffSize <- sort(confIntEffSize)
     }
+    if (optionsList$wantsEffSizeSE){
+      effectSizeVar <- ((1/n)+(d**2 / (2*n))) * (2*(1-cor(c1,c2)))
```

Comment:
```suggestion
      effectSizeVar <- ((1/n)+(d^2 / (2*n))) * (2*(1-cor(c1,c2)))
```
use `^` rather than `**` (throughout)